src/core/app_backend.tf

```diff
@@ -79,6 +81,7 @@ locals {
       IO_SIGN_API_BASE_PATH             = "/api/v1/sign"
       LOLLIPOP_API_BASE_PATH            = "/api/v1"
       TRIAL_SYSTEM_API_BASE_PATH        = "/api/v1"
+      IO_WALLET_API_BASE_PATH           = "/api/v1"
```

Comment:
```suggestion
      IO_WALLET_API_BASE_PATH           = "/api/v1/wallet"
```

Reply:
do we want to add the wallet path?

Reply:
I think it is necessary otherwise you would have too common endpoints that could also be used by other domains. Also in accordance with what is stated [here](https://github.com/pagopa/io-backend/blob/48fd0fc4cedbd8ad1b3575f9a9e3bceb60ea56f6/api_io_wallet.yaml#L6)

Reply:
No, I got confused with the io-backend path.